cmd/lakefs/cmd/run.go

```diff
@@ -531,6 +536,17 @@ func enableKVParamsMetrics(p params.Config) params.Config {
 	return p
 }
 
+func initOTFService() *diff.Service {
+	pluginsManager := plugins.NewManager[diff.Differ]()
+	pid := plugins.PluginIdentity{Version: 0, ExecutableLocation: "~/.lakefs/plugins/delta"}
+	pa := plugins.PluginAuth{Key: "deltaKey", Value: "7a44968c-5076-4a90-9a0e-0249cc6c119a"}
```

Comment:
1. Please add a comment how to find / generate this value.
1. Could it be just a readable string such as "kind: diff; type: delta; version: 0.0.1" (I don't have any particular preference for the format to use!)?

Reply:
1. Adding the location to the design. Will be updated in the final version to have 
2. It would include data duplication if it would be of this format.